src/deprecation.h

```diff
@@ -10,7 +10,7 @@
 // Per https://zips.z.cash/zip-0200
 // Shut down nodes running this version of code, 16 weeks' worth of blocks after the estimated
 // release block height. A warning is shown during the 14 days' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 1022000;
+static const int APPROX_RELEASE_HEIGHT = 1028431;
```

Comment:
the current mainnet block height is 1028900, which is greater than this release height. i suggest a release height ~24 hours from now: 1028900 + 24 * 3600 / 75 ~= 1030050

```suggestion
static const int APPROX_RELEASE_HEIGHT = 1030050;
```